com/AlterETM.py

```diff
@@ -87,8 +87,8 @@ def print_params(cnn, stnlist):
                                      % (station['NetworkCode'], station['StationCode']), as_dict=True)
 
             for p in params:
-                print(' %s.%s %-5s %-5s %2i' \
-                      % (station['NetworkCode'], station['StationCode'], p['soln'], p['object'], p['terms']))
+                print(' %s.%s %-5s %-5s %2i' % (station['NetworkCode'], station['StationCode'],
```

Comment:
how about:
```python
print(' %s.%s %-5s %-5s %2i' % (station['NetworkCode'],
                                station['StationCode'],
                                p['soln'], p['object'],
                                p['terms']))
```
so we stay under the 79 character length?

Reply:
Is 80 length today useful? Modern editor could more.
In PR #109 in .github/workflows/python-package-conda.yml line 29+30 is set to --max-line-length=127       
```
 # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
```

Reply:
Yes, most projects still use a 79 character limit-- the character length convention is partly so that multiple editor windows can be opened side by side for comparing files.

Right now, #109 is just running flake8 as warnings, but there will be another github runner automation run for new Pull Requests that won't pass with pep8 errors, and will likely use the 79 character limit.

Reply:
Thank you, have a question about the Output Formatting, what code style you preferred?
- Old-style String Formatting (% operator): This method is less preferred in newer Python code but still works.
- String Formatting (.format() method)
- Formatted String Literals (f-strings) (Python 3.6+)

Reply:
My strong preference and vote is for us to use Formatted String Literals (f-strings) moving forward. 

We are currently pinned to using sklearn 1.1 or newer (which requires Python 3.8+)... and even if remove that dependency we are still required to use numpy version 1.19.5 or newer (which requires Python 3.6+). So we should always be on Python 3.6+ and have f-strings available as part of the standard library.

Reply:
That sounds this line would change in the future.
If it possible say ok to  this review and accept the PR? The other lines you requested, I make no change in that line, only a line upper.